app/data/1_0001.json

```diff
@@ -0,0 +1,5127 @@
+{
+    "data_version": "0.0.2",
+    "description": "UK Innovation Survey",
+    "groups": [
+        {
+            "blocks": [
+                {
+                    "id": "general-business-information-block-3",
```

Comment:
The id's throughout the survey are not great, there is no reason for 'block' to appear in this id. If there are multiple blocks that need to be called the same thing they should start at 1, so this block id would be `general-business-information-1`